maven-release-plugin/src/main/java/org/apache/maven/plugins/release/PerformReleaseMojo.java

```diff
@@ -46,7 +46,7 @@
  */
 @Mojo( name = "perform", aggregator = true, requiresProject = false )
 public class PerformReleaseMojo
-    extends AbstractReleaseMojo
+    extends AbstractScmReleaseMojo
```

Comment:
see https://github.com/apache/maven-release/pull/145#issuecomment-1221943545. This exposes a lot more unused parameters, not just tag

Reply:
@kwin I see your point. What's the best course of action? Wait for 145 to be resolved, implement the changes suggested by 145, or just add tag property to perform release mojo?

Reply:
@michael-o Please advise how to continue.

Reply:
@kwin I see you commented on #145 to create 2 separate classes. I believe if they are created, this pull request may be made redundant